### PR TITLE
Bump unpkg example to 3.0.0

### PR DESCRIPTION
**Pull request by rgbkrk**
I copy pasted the unpkg css lines for blueprintjs while working with the 3.x release then finally noticed that the referenced css was the 2.x line. Not sure if the code snippet below this needs it.

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @rgbkrk! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/docs-app/src/getting-started.md [hunk 1]
```diff
@@ -92,8 +92,8 @@ These bundles _do not include_ external dependencies; your application will need
     <script src="https://unpkg.com/react-transition-group@^2.2.1/dist/react-transition-group.min.js"></script>
     <script src="https://unpkg.com/popper.js@^1.12.6/dist/umd/popper.js"></script>
     <script src="https://unpkg.com/react-popper@~0.7.4/dist/react-popper.min.js"></script>
```
Comment (giladgray) [inline packages/docs-app/src/getting-started.md]: popper.js and react-popper versions need to change too.

https://github.com/palantir/blueprint/blob/develop/packages/core/package.json#L46-L47

Comment (rgbkrk) [reply packages/docs-app/src/getting-started.md]: Happy to!

**Comment by rgbkrk** [comment]
screen shot 2018-07-15 at 10 00 35 am

**Comment by rgbkrk** [comment]
Tracing through the built dist, I'm seeing where `Gn` ends up undefined:

https://gist.github.com/rgbkrk/e3c3770def1c2e0a554d572f3e6e5052#file-blueprint-core-dist-js-L2013-L2035

Not real sure why that is.

File: packages/docs-app/src/getting-started.md [hunk 2]
```diff
@@ -81,19 +81,19 @@ These bundles _do not include_ external dependencies; your application will need
...
     <script src="https://unpkg.com/dom4@^1.8"></script>
     <script src="https://unpkg.com/react@^16.2.0/umd/react.production.min.js"></script>
     <script src="https://unpkg.com/react-dom@^16.2.0/umd/react-dom.production.min.js"></script>
     <script src="https://unpkg.com/react-transition-group@^2.2.1/dist/react-transition-group.min.js"></script>
-    <script src="https://unpkg.com/popper.js@^1.12.6/dist/umd/popper.js"></script>
-    <script src="https://unpkg.com/react-popper@~0.7.4/dist/react-popper.min.js"></script>
-    <script src="https://unpkg.com/@blueprintjs/core@^2.0.0"></script>
-    <script src="https://unpkg.com/@blueprintjs/icons@^2.0.0"></script>
+    <script src="https://unpkg.com/popper.js@^1.14.1/dist/umd/popper.js"></script>
+    <script src="https://unpkg.com/react-popper@1.0.0/dist/index.umd.min.js"></script>
```
Comment (giladgray) [inline packages/docs-app/src/getting-started.md]: this should be `^1.0.0`

**Comment by giladgray** [comment]
@rgbkrk i don't see that error. is it still a problem on your end?

**Comment by rgbkrk** [comment]
Yeah I took this example as part of a file called `test.html`. I'll try it again soon.

**Pull request merged**